src/OVAL/probes/SEAP/generic/rbt/rbt_i64.c

```diff
@@ -51,7 +51,14 @@ static struct rbt_node *rbt_i64_node_alloc(void)
 static void rbt_i64_node_free(struct rbt_node *n)
 {
         if (n != NULL)
+#ifndef _WIN32
                 free(rbt_node_ptr(n));
+#else
+                // using free for memory allocated through _aligned_malloc is illegal
+                // rbt_i64.c -> rbt_i64_node_alloc
+                // https://msdn.microsoft.com/en-us/library/8z34s9c6.aspx
+                _aligned_free(rbt_node_ptr(n));
```

Comment:
@mpreisler I just followed the current identation, should I make a new identation in the file?